.woodpecker.yml

```diff
@@ -6,7 +6,8 @@ pipeline:
   check_ansible_format:
     image: alpine:3
     commands:
-      - apk add ansible
+      - apk add ansible ansible-lint
       - ansible-playbook lemmy.yml --syntax-check
       - ansible-playbook lemmy-almalinux.yml --syntax-check
       - ansible-playbook uninstall.yml --syntax-check
+      - ansible-lint --warn-list experimental lemmy.yml lemmy-almalinux.yml uninstall.yml
```

Comment:
You can consider making separate steps for these so its easier to tell whats failing. But like this is also okay.

Reply:
Not directly related to this PR, but have you considered using GitHub actions?
`ansible-lint` can produce [SARIF](https://sarifweb.azurewebsites.net/) output, which [GitHub can use for annotations directly in the diff](https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning).
As far as I can tell, third party CIs are not able to do this.

Reply:
I've moved it to a separate woodpecker pipeline now.

Reply:
We want to avoid relying on another proprietary service. Woodpecker is selfhosted and fully open source so its a better match for Lemmy.